web/src/components/SubscriptionSettings/SubscriptionSettings.js

```diff
@@ -0,0 +1,56 @@
+import { useMutation } from '@redwoodjs/web'
+import { useState } from 'react'
+import { VscLoading } from 'react-icons/Vsc'
+import { useForm } from 'react-hook-form'
+import { Form, Submit } from '@redwoodjs/forms'
+import { navigate, Redirect, routes } from '@redwoodjs/router'
+
+const SUBSCRIBE_USER = gql`
+  mutation createSessionMutation($input: CreateCheckoutSessionInput!) {
+    createCheckoutSession(input: $input)
+  }
+`
+const SubscriptionSettings = () => {
+  const [subscribe, { loading }] = useMutation(SUBSCRIBE_USER)
+  const [subscribed, setSubscribed] = useState('Subscribe Now')
+  const formMethods = useForm()
+
+  if (loading) {
+    return (
+      <VscLoading className="animate-spin text-8xl text-green-700 mx-auto h-screen" />
+    )
+  }
+  const onSubmit = async () => {
+    const testMonthlyPriceId = 'price_1JtImLL0VIDl9kuolU9h47jP'
+    const liveMontlyPriceId = 'price_1Jy2WHL0VIDl9kuouIAEJNyC'
```

Comment:
Can we use environment variables for these? In the prod env we can use the live price ID, in dev the test one, etc.

Reply:
Also there's a typo in `liveMontlyPriceId`. 😛